subprojects/docs/src/docs/userguide/javaPlugin.adoc

```diff
@@ -799,6 +793,20 @@ In this way, you can use `gradle test` to test JUnit 3/4 tests on JUnit Platform
 
 A sample of mixed tests can be found at `samples/testing/junitplatform/engine` in the '-all' distribution of Gradle.
 
+[[custom_test_engine]]
+===== Custom test engine
+
+With JUnit Platform, you can customize `TestEngine` implementation yourself. By default, `TestEngine` registration is done via Java's `java.util.ServiceLoader` mechanism.
```

Comment:
JUnit platform allows you to use different test engines.

Reply:
Done.